quickstarts/101-hello-power-platform/main.tf

```diff
@@ -9,14 +9,14 @@ terraform {
     null = {
       source = "hashicorp/null"
     }
-    power-platform = {
+    powerplatform = {
```

Comment:
hi @mawasile why do we want to revert?  In the Pubic remo and in the provider doc's we have examples and documentation with **"power-platform"** like this:

`terraform {
  required_providers {
    power-platform = {
      source = "microsoft/power-platform"
      version = "2.0.2-preview"
    }
  }
}

provider "power-platform" {
  # Configuration options
}`

Reply:
the examples in registry are coming from this repo. Secondly, the "-" doesn't seem to work (anymore?)